.drone/drone.jsonnet

```
@@ -345,7 +345,23 @@ local manifest(apps) = pipeline('manifest') {
     steps: [
       make('check-drone-drift', container=false) { depends_on: ['clone'] },
       make('check-generated-files', container=false) { depends_on: ['clone'] },
-      make('test', container=false) { depends_on: ['clone', 'check-generated-files'] },
+      make('test', container=false) { depends_on: ['clone'] },
+      run('clone-main', commands=['cd ..', 'git clone $CI_REPO_REMOTE loki-main', 'cd -']),
+      run('test-main', commands=['cd ../loki-main', 'BUILD_IN_CONTAINER=false make test']) { depends_on: ['clone-main'] },
+      make('compare-coverage', container=false, args=[
+        'old=../loki-main/test_results.txt',
+        'new=test_results.txt',
+        'packages=ingester,distributor,querier,ruler,storage',
```

```suggestion
        'packages=ingester,distributor,querier,iter,storage',
```

The ruler is not super critical here, but the `iter` package has far-reaching consequences.
Maybe @cyriltovena can suggest any other critical packages we need to cover if there are any?

---

I changed the list to these: `ingester,distributor,querier,iter,storage,chunkenc,logql,loki`

I think we can be fairly aggressive here because we ought to have color coding to help us see where to focus.

---

does it fail if the package name has changed ?

---

We'd see a large diff, but this won't fail